### DISCUSSION TITLE
Welcome to pytorch-deep-learning Discussions!

### DISCUSSION THREAD

**Discussion by mrdbourke** [mrdbourke/pytorch-deep-learning#2] [Announcements]
Yo yo!

If you have any questions or ideas, please leave a discussion so we can jam on it.

Let's have some fun.

**Comment by prdai** [comment]
Looking forward for this course :)

Keep up the good work

**Comment by prdai** [comment]
5.0 Transfer Learning Fine-Tunning Freezing Layers, etc...

**Comment by fivefishstudios** [comment]
After learning OpenCV, I wanted more and decided to learn PyTorch next and I just want to say thank you for all the hard work and time you've put into this. This has given me a solid plan of attack on where to start/how to learn PyTorch. Thank you!
